dev.html

```diff
@@ -70,6 +70,7 @@ <h2 class="title" id="join">Join us!</h2>
 <h2 class="title page-title text-center" id="projects">Our Projects</h2>
 <div class="dev-projects-grid">
     {% for proj in site.data.projects %}
+    {% if proj.wip != "true" %}
```

Comment:
This is a little nitpicky, but I think Liquid's `where` filter is quite useful here. You can read about it more on the [Liquid docs](https://shopify.github.io/liquid/filters/where/) or by looking at the entry in the [Jekyll docs](https://jekyllrb.com/docs/liquid/filters/). I'm thinking something like this should work (but not 100% sure):

```
{% assign finished = site.data.projects | where:"wip" %}
{% for proj in finished %}
```

If this doesn't work / it's a bit of a head-scratcher, what you have now works as-is!